creativecloud/blocks/interactive-metadata/interactive-metadata.css

```diff
@@ -0,0 +1,398 @@
+
+:root {
+  --prompt-input-fill: #262626;
+  --prompt-btn-fill: #323232;
+  --prompt-text-color: #D1D1D1;
+  --prompt-text-border: #545454;
+  --prompt-btn-text: #FFFFFF;
+  --prompt-highlight-color: #1473E6;
+  --layer-background-fill: #272928;
+  --prompt-btn-fill-light: #ffff;
```

Comment:
Can we make it upercase as rest of the color code?

Reply:
Made the suggested change